db/models.py

```diff
@@ -46,3 +50,73 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE
+    )
+
+    class Meta:
+        ordering = ["-created_at"]
+
+    def __str__(self) -> str:
+        return f"{self.created_at}"
+
+
+class Ticket(models.Model):
+    movie_session = models.ForeignKey(
+        MovieSession,
+        on_delete=models.CASCADE
+    )
+    order = models.ForeignKey(
+        Order,
+        on_delete=models.CASCADE
+    )
+    row = models.IntegerField()
+    seat = models.IntegerField()
+
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(fields=["row", "seat", "movie_session"],
+                                    name="unique_row_seat_session")
+        ]
+
+    def __str__(self) -> str:
+        return (f"{self.movie_session.movie.title} "
+                f"{self.movie_session.show_time} "
+                f"(row: {self.row}, seat: {self.seat})")
+
+    def clean(self) -> None:
+        if not 1 < self.row <= self.movie_session.cinema_hall.rows:
+            raise ValidationError({
+                "row": [
+                    "row number must be in available range: "
+                    "(1, rows): "
+                    f"(1, {self.movie_session.cinema_hall.rows})"
+                ]
+            })
+        if not 1 < self.seat <= self.movie_session.cinema_hall.seats_in_row:
```

Comment:
I know it passes test, but i"m pretty sure, that seat with number 1 could be in there

Reply:
Here agree)